Resources/Dockerfiles/neo4jescodb.Dockerfile

```diff
@@ -2,20 +2,38 @@
 
 #APOC plugin version must track the major.minor version of neo4j so the base image for this dockerfile is specified rather than using latest
 #https://github.com/neo4j-contrib/neo4j-apoc-procedures#version-compatibility-matrix
-FROM neo4j:3.5
+FROM neo4j:4.0.4
 
 #the esco file must be provisioned in a folder that is mounted to /var/lib/neo4j/import/
 ENV ESCOFILE esco_v1.0.3.ttl
 ENV NEO4J_PWD escodb
 ENV NEO4J_AUTH neo4j/$NEO4J_PWD
+ENV NEO4J_dbms_allow__upgrade=true
+ENV NEO4J_dbms_recovery_fail__on__missing__files=false
 ENV NEO4J_dbms_memory_pagecache_size=5600M
+ENV NEO4J_dbms_connector_bolt_address=0.0.0.0:7687
+ENV NEO4J_dbms_connector_bolt_enabled=true
+ENV NEO4J_dbms_connector_bolt_tls__level=OPTIONAL
+ENV NEO4J_dbms_connector_bolt_listen__address=0.0.0.0:7687
+ENV NEO4J_dbms_connector_http_enabled=true
```

Comment:
Can we disable http please?
Even though there's no sensitive data in the graph, we want to avoid people potentially sniffing tokens/credentials.

Reply:
removed http support from docker and disabled http ports on container level.